products/opensuse/main.pm

```diff
@@ -67,7 +67,7 @@ sub cleanup_needles() {
     if (!get_var('LEAP')) {
```

Comment:
Not touched by your patch - but for consistent style it might be worthy to use if (!is_leap) here